### PR TITLE
CVSL-444: Record digital signature of variation approver

### PR DESCRIPTION
**Pull request by smcveigh941**

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt [hunk 1]
```diff
@@ -683,6 +684,9 @@ class LicenceService(
       statusCode = ACTIVE,
       dateLastUpdated = LocalDateTime.now(),
       updatedByUsername = username,
+      approvedByUsername = username,
+      approvedDate = LocalDateTime.now(),
+      approvedByName = body.fullName
     )
```
Comment (timharrison-moj) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:690]: Should the full name of the approver also go into the events below this? Looking at it, its assuming that the approver will also appear in the comDetail table - recorded at login, so it looks like the full name is already obtainable from there?

**Review by timharrison-moj — COMMENTED** [review]
LGTM. One question about whether the additional body content of full name is necessary, or already obtainable from comDetail?

File: src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt (hunk 1 above)
Comment (smcveigh941) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt:689]: Sorted this now, thanks

**Review by timharrison-moj — APPROVED** [review]
LGTM

**Pull request merged**